### PR TITLE
Refactor rake task to make it more obvious what is happening.

### PR DESCRIPTION
**Pull request by troessner**

### CODE REVIEW

File: Rakefile [hunk 1]
```diff
@@ -3,6 +3,20 @@ require 'rake/clean'
 
 Dir['tasks/**/*.rake'].each { |t| load t }
 
-task local_test_run: [:test, :rubocop, 'test:quality']
-task ci: [:test, :rubocop, 'test:quality', :ataru]
+defaults = ['test:spec',
+            # Should run before test:features so we have the the most up-to-date default configuration
```
Comment (mvz) [inline Rakefile]: This is no longer needed since the `defaults.reek.yml` configuration file is only for documentation now.

Comment (troessner) [reply Rakefile]: Good point, removed the comment.

File: Rakefile [hunk 2]
```diff
@@ -3,6 +3,20 @@ require 'rake/clean'
...
+
+task :local_test_run do
+  defaults.each do |name|
+    puts "\n=== Running #{name}...\n"
+    Rake::Task[name].invoke
+    puts "\n=== Running #{name} -> Done\n"
+  end
+end
+
+task ci: (defaults + [:ataru])
```
[the remark this answers is not in the document]
Comment (troessner) [reply Rakefile]: Yeah, i think it was for some hyperoptimization in regards to speed 😆 

```
time bundle exec rake ataru
Run options: --seed 57949

# Running:

.string -- 6 warnings:
  IrresponsibleModule: Dirty has no descriptive comment
  UncommunicativeMethodName: Dirty#m has the name 'm'
  UncommunicativeParameterName: Dirty#m has the parameter name 'a'
  UncommunicativeParameterName: Dirty#m has the parameter name 'b'
  UncommunicativeParameterName: Dirty#m has the parameter name 'c'
  UnusedParameters: Dirty#m has unused parameter 'c'
.has no descriptive comment
has the name 'C'
.......

Finished in 0.015756s, 571.2209 runs/s, 0.0000 assertions/s.

9 runs, 0 assertions, 0 failures, 0 errors, 0 skips

real	0m2.345s
user	0m1.661s
sys	0m0.447s
```

Obviously this doesn't make much sense. What do you think about just having the "ci" task and point the "default" to it?

[the remark this answers is not in the document]
Comment (mvz) [reply Rakefile]: Yes, that makes sense.

**Pull request merged**